dozer/bot.py

```diff
@@ -48,7 +48,8 @@ def __init__(self, config: dict, *args, **kwargs):
 
     async def setup_hook(self) -> None:
         for ext in os.listdir('dozer/cogs'):
-            if not ext.startswith(('_', '.')):
+            cog_name = ext[:-3]
+            if not ext.startswith(('_', '.')) and ext.endswith(".py") and not cog_name in self.config.get("disabled_cogs", []):
                 await self.load_extension('dozer.cogs.' + ext[:-3])  # Remove '.py'
```

Comment:
NIT: If we're making the `cog_name` variable, it simplifies things to use that variable on this line too.